wrapper/src/main/java/software/amazon/jdbc/hostlistprovider/RdsHostListProvider.java

```diff
@@ -197,7 +197,7 @@ protected void init() throws SQLException {
           this.isPrimaryClusterId = clusterSuggestedResult.isPrimaryClusterId;
         } else {
           final String clusterRdsHostUrl =
-              rdsHelper.getRdsClusterHostUrl(this.initialHostSpec.getHostAndPort());
```

Comment:
so this broke 2.3.8, 2.3.9 and 2.4.0 as per https://github.com/aws/aws-advanced-jdbc-wrapper/issues/1138 , maybe you should add a test ?

Reply:
Test added, thanks